code/control/DeployPlanDispatcher.php

```diff
@@ -67,6 +68,21 @@ public function index(\SS_HTTPRequest $request) {
 
 	/**
 	 * @param SS_HTTPRequest $request
+	 * @return SS_HTTPResponse
+	 */
+	public function gitupdate(SS_HTTPRequest $request) {
+		switch($request->httpMethod()) {
+			case 'POST':
+				return $this->createFetch();
```

Comment:
Can anyone who can see the project fetch? Should we be introducing a fetch-specific "allow" code?